### PR TITLE
Update COVID guidance before the upcoming techday

### PR DESCRIPTION
**Pull request by Scarzy**
In the absence of information from the venues, and recognising that we need something in place by the tech day this weekend, I suggest we add this COVID policy. I am expecting it to change to some degree before the competition.

### CODE REVIEW

File: covid-19.md [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
 ---
 
 We will keep this page updated with new information as it becomes available.
 
 Last updated: {{ page.date | date: "%e %B %Y" }}.
 
+1. If you are experiencing symptoms of COVID-19, or have tested positive for COVID-19 in the 7 days beforehand (or live with someone who has) please do not attend.
+2. If you are able, please wear a mask when not in your team pit.
+3. Please clean your hands regularly, using hand sanitizer or soap.
+4. Be respectful towards others personal space.
```
Comment (PeterJCLaw) [inline covid-19.md:14]: nit:
```suggestion
4. Be respectful towards others' personal space.
```

File: covid-19.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
 ---
 layout: blog
 title: COVID-19 Updates
-date: 2021-12-08
+date: 2022-03-31
 ---
 
 We will keep this page updated with new information as it becomes available.
 
 Last updated: {{ page.date | date: "%e %B %Y" }}.
```
Comment (PeterJCLaw) [inline covid-19.md:10]: Should we have a title or something here, so that it doesn't just launch straight into the list?
Even just `## Events` might do?

Comment (Scarzy) [reply covid-19.md:10]: I'll add some prose

File: covid-19.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Scarzy) [reply covid-19.md]: Do you have a screenshot?

File: covid-19.md (hunk 2 above)
Comment (Scarzy) [reply covid-19.md:10]: I changed most of the wording really...

File: covid-19.md [hunk 3]
```diff
@@ -1,15 +1,26 @@
...
+## SR2022
 
-It is currently our intention to run an in person competition for the SR2022 competition cycle. This is of course dependent on the government guidelines which we are keeping a close eye on. If you have any COVID-19 specific questions about the competition please contact us at [{{ site.emails.teams }}][teams-email].
+Life is beginning to get back to normal, and this means we are finally able to do in person events again!
+However, we must recognise that some people still have to be cautious, to protect their own health or their friends or family.
+As such, we ask that everyone follows the below requests to help ensure we all keep safe, whilst still having lots of fun with robots!
+
+* If you are experiencing symptoms of COVID-19, or have tested positive for COVID-19 in the 7 days beforehand (or live with someone who has) please do not attend.
+* If you are able, please wear a mask when not in your team pit.
+* Please clean your hands regularly, using hand sanitizer or soap.
```
Comment (RealOrangeOne) [inline covid-19.md:19]: Worth mentioning that we won't be providing sanitizer?

Comment (Scarzy) [reply covid-19.md:19]: I thought about this.

I'm largely still debating whether we need to provide some or not (at least for the main competition). However that's beyond the scope of this PR.

I can't say with any confidence whether or not the venue for either the techday or the competition will have hand sanitizer available anyway.
It is reasonable to assume the toilets will have some kind of soap or sanitizer in them, which means there is always the soap fallback.
Some young people can, at times, be incredibly pedantic and might use any kind of inaccuracy to justify not complying.
A number of people will carry their own anyway.
So it's easier to just not mention it.

**Pull request merged**